substrate-node/pallets/pallet-dao/src/lib.rs

```diff
@@ -506,6 +508,12 @@ impl<T: Config> Pallet<T> {
         let su = resources::get_su(resources);
         cu * 2 + su
     }
+
+    fn is_council_member(who: T::AccountId) -> bool {
```

Comment:
Make the function return a `DispatchResultWithPostInfo` and move the `ensure!(..)` check in this function.